cppapi/client/filedatabase.h

```diff
@@ -199,17 +199,12 @@ class FileDatabase
 	static int MaxWordLength;
 
 
-	int 			length_buf;
-  	int 			pos_buf;
   	int 			CrtLine;
   	int 			StartLine;
   	char 			CurrentChar;
   	char 			NextChar;
 
-  	bool 			DELETE_ENTRY;
   	std::string 			word;
-
-	FileDatabaseExt	*ext;
```

Comment:
The ext data member should be kept here because it is there to help us to extend this class without breaking the binary compatibility in the future.
For this version, I think it is fine to remove the other fields because we will break the binary compatibility anyway in the next version but I think we should keep the ext data member.

By the way, this reminds me that since the next version will not be binary compatible with 9.3.3, we should take this opportunity to move all the features (data members/ methods) which have been implemented in some ext classes into their corresponding Tango class.

Reply:
Done